### PR TITLE
fix issue with missing nslookup ADDON-38627

### PR DESCRIPTION
**Pull request by aryznar-splunk**

### CODE REVIEW

**Comment by aryznar-splunk** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by aryznar-splunk** [comment]
\@aryznar-splunk

**Comment by aryznar-splunk** [comment]
CLA Assistant Lite bot CLA Assistant Lite bot All contributors have signed the COC ✍️ ✅

**Comment by aryznar-splunk** [comment]
I have read the Code of Conduct and I hereby accept the Terms

File: deploy/install_full.bash [hunk 1]
```diff
@@ -54,22 +54,24 @@ verify_dependencies() {
...
 
-install_snapd_on_centos7() {
+install_dependencies_on_centos7() {
   yum -y install epel-release
   yum -y install snapd
   systemctl enable snapd
   systemctl start snapd
   sleep 10
   ln -s /var/lib/snapd/snap /snap
+  yum -y install bind-utils
```
Comment (lstoppa) [inline deploy/install_full.bash:64]: I think this is only part of the general solution.
https://www.hostwinds.com/tutorials/installing-bind-centos-7

I have not tested this, however it looks like you need to:
systemctl enable named
systemctl start named

File: deploy/install_full.bash [hunk 2]
```diff
@@ -54,22 +54,24 @@ verify_dependencies() {
...
 
-install_snapd_on_centos8() {
+install_dependencies_on_centos8() {
   dnf -y install epel-release
   dnf -y install snapd
   systemctl enable snapd
   systemctl start snapd
   sleep 10
   ln -s /var/lib/snapd/snap /snap
+  yum -y install bind-utils
```
Comment (lstoppa) [inline deploy/install_full.bash:74]: see above

File: deploy/install_full.bash (hunk 1 above)
Comment (aryznar-splunk) [reply deploy/install_full.bash:64]: No I dont need as nslookup is command not service as snmpd

File: deploy/install_full.bash (hunk 2 above)
Comment (aryznar-splunk) [reply deploy/install_full.bash:74]: No I dont need as nslookup is command not service as snmpd

**Comment by aryznar-splunk** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Review by lstoppa — APPROVED** [review]

**Pull request merged**